apps/drupal/templates/block/block--basic--card.html.twig

```diff
@@ -25,7 +25,7 @@
  * @see template_preprocess_block()
  *
  * @See patternlab_theme_suggestions_block_alter()
- * @see @molecules/_card.twig for full list of variables.
+ * @see @default_molecules/_card.twig for full list of variables.
```

Comment:
Interesting comment on 'bootstrap_` as a prefix over `default_`. 

I think the simplest change is to with "default" for now, and the Tailwind implementation could see us renaming things so we have a `source/bootstrap_default` and `source/tailwind_default` directory... I like the thought... Also this plays into one of my goals, which is the `tools/new-design-system` tool. So we likely wouldn't have ANYthing in `source` by default, but run a generator where we'd select all the defaults we want, including all the protons/atoms/orgs/etc.

Reply:
Revisiting this:
The update to component library module will provide _named_ prefixing for namespaces in Drupal. This means that we can prefix any of Drupal namespaces with the Theme name or Module name.  So that's going to be some peanut butter and jelly to work for namespacing between Design System + Theme/Module.

Consider the following scenario:

Theme 1: Particle
Theme 2: Barnacle
Module 1: Cats

Drupal will look for the following namespaces:

* `particle_atoms`
* `particle_molecules`
* `particle_organisms`

* `barnacle_atoms`
* `barnacle_molecules`
* `barnacle_organisms`

* `cats_atoms`
* `cats_molecules`
* `cats_organisms`

Naming imports `default_atoms` in this scenario will be breaking unless we update namespace maker to do some more advanced reading. All that to say, is there a scenario where we're not integrating One Design System to One Theme implementation? In the Drupal scenario above I think we can gain a lot of benefit from having a 1:1 relationship. Namespaces are shared between component libraries as long as the module or theme instance is installed. Which is a nicety when you consider multiple themes, admin themes, modules using theme resources, etc.

Reply:
@jryanconklin There is a use case we have for a client where one design system would feed two drupal themes. 

I don't think Components should be declaring what our namespaces SHOULD be are they?

So if I know my design system is going to be used for a Drupal theme too, then inside my normal design system I should be able to declare my `atoms` to be `default_atoms` or `super_secret_atoms` if I choose, and map it to any directory under `_patterns`. 

In that scenario, our patterns and twig files would be using `@import default_atoms`, etc. so would our drupal theme have to instead use `@import particle_default_atoms` for Components to know what is what?? Let's chat this one out. I'm really confused here maybe.

Reply:
All I'm thinking is I shouldn't have to (or the new system tool shouldn't have to) limit the format of namespaces based on a Drupal module.

Reply:
For sure! The change I made to Components allows for prefixing by the name as an option. So in the above scenario, the sets array will write out to info.yml as `default_atoms`. With prefixing on, that'd be transformed to `themeName_default_atoms`. 

Practically speaking, I don't think it's necessarily a limitation since it's an option in the module. Though I do still think it makes more sense to name things 1:1. There's nothing stopping us from creating a "base" design system to take place of the default idea.